nexus/src/external_api/http_entrypoints.rs

```diff
@@ -731,6 +737,134 @@ async fn saml_identity_provider_view(
 
 // TODO: no DELETE for identity providers?
 
+// "Local" Identity Provider
+
+/// List users in the given local identity provider
+#[endpoint {
+    method = GET,
+    path = "/system/silos/{silo_name}/identity-providers/local/users/all",
```

Comment:
I'm going by [RFD 321](https://github.com/oxidecomputer/rfd/tree/master/rfd/0321#changes-to-identity-provider-api-routes-1).  The reason I proposed it that way in RFD 321 is that I wanted to put other things under `.../users`.  Currently that's `.../users/id/:id` (see your other comment below), and it could later include `.../users/external_id/:external_id` or `.../users/bulk_fetch` (see #1209).  Technically this would be okay provided that none of these other path components could be confused for a uuid, but I hate relying on that.  It felt cleaner to separate the collection of users from the other endpoints.

I imagine much of this will wind up reworked under RFD 322.

Reply:
:+1: